src/components/Settings.svelte

```diff
@@ -1,20 +1,58 @@
 <script>
+  import { beforeUpdate, afterUpdate } from 'svelte';
   import { Tabs, Tab, TabContent, MaterialApp } from 'svelte-materialify/src';
   import UISettings from './settings/UISettings.svelte';
   import FilterSettings from './settings/FilterSettings.svelte';
 
   export let isStandalone = false;
+  export let isResizing = false;
 
   const settings = [
     { name: 'Interface', component: UISettings },
     { name: 'Filters', component: FilterSettings }
   ];
+
+  function redrawSlider() {
+    const sliderElement = document.querySelector('.ltl-settings-slider');
+    const activeTab = 
+      document.querySelector('.ltl-settings-tabs .s-tab.s-slide-item.active');
+    if (
+      !sliderElement || 
+      !activeTab || 
+      !sliderElement.offsetParent || 
+      !activeTab.offsetParent) return;
+
+    sliderElement.style.left = `${activeTab.offsetLeft}px`;
+    sliderElement.style.width = `${activeTab.offsetWidth}px`;
+  }
+
+  let callRedrawSlider = false;
+  let wasResizing = false;
+  $: wasResizing = !isResizing;
+  beforeUpdate(() => {
+    if (wasResizing) {
+      callRedrawSlider = true;
+    }
+  });
+  afterUpdate(() => {
+    if (callRedrawSlider) {
+      redrawSlider()
+    }
+    callRedrawSlider = false;
+  });
+  window.onresize = redrawSlider;
```

Comment:
Can you change this to add the event listener on mount and remove it when the component is destroyed? I think either using onMount and onDestroy or using the svelte:window component will work.

Reply:
done

Reply:
Done on latest commit

Reply:
modified and done in the latest